### PR TITLE
chore: remove duplicate scss

### PR DESCRIPTION
**Pull request by harish-sethuraman**
Replacement of https://github.com/eslint/eslint.org/pull/365

Completely verified the changes locally and checked cascading of rules before and after removing duplicate selectors

### CODE REVIEW

File: src/assets/scss/forms.scss [hunk 1]
```diff
@@ -361,10 +301,6 @@ ul.selected-options {
     outline-offset: 3px;
 }
 
-.c-checkbox input[type="checkbox"]:focus ~ .c-checkbox__icon {
-    outline: 3px solid transparent;
-}
```
Comment (kecrily) [inline src/assets/scss/forms.scss:366]: There is a difference with the reserved line 360 line, which is 2px and here is 3px, but it is not a big problem.

File: src/assets/scss/forms.scss [hunk 2]
```diff
@@ -1,61 +1,3 @@
...
-    background-size: 1em auto, 100%;
-}
-
-.label__text.label__text {
-    display: flex;
-    align-items: center;
-    gap: 0.5rem;
-    font-size: 0.875rem;
-    font-family: var(--text-font);
-    color: inherit;
```
Comment (kecrily) [inline src/assets/scss/forms.scss:39]: There is a difference from the reserved line 121, which is `color: var(--color-neutral-700);`, but I think the reserved one is the expected behavior.

**Review by kecrily — APPROVED** [review]
LGBT

File: src/assets/scss/forms.scss (hunk 1 above)
Comment (harish-sethuraman) [reply src/assets/scss/forms.scss:366]: Seemed like 2px was being applied the whole time so removed the 3px one

File: src/assets/scss/forms.scss (hunk 2 above)
Comment (harish-sethuraman) [reply src/assets/scss/forms.scss:39]: Yea this was getting over written by the above one so removed it

**Pull request merged**